webpack.config.js

```diff
@@ -24,6 +24,10 @@ module.exports = {
   plugins: [
     new BundleTracker({ filename: './webpack-stats.json' }),
     new ExtractTextPlugin('[name]-[hash].css'),
+    new webpack.ProvidePlugin({
+      'Promise': 'exports?global.Promise!es6-promise',
```

Comment:
I don't know it for sure, but native promise is WAY OLD.  (29!)

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise